test-support/helpers/sl/utils/utils.js

```diff
@@ -3,30 +3,30 @@ import Ember from 'ember';
 /**
  * Converts provided parameter into an array
  *
- * @function convertToArray
- * @param   {array|string|object} underTest
- * @throws  {Ember.assert} If not provided an Array, String or Object
- * @returns {array}
+ * @function
+ * @param {Array|String|Object} underTest
+ * @throws {Ember.assert} If not provided an Array, String or Object
+ * @returns {Array}
  */
-var convertToArray = function( underTest ) {
-    var returnArray;
+let convertToArray = function( underTest ) {
+    let returnArray;
 
     if ( Array.isArray( underTest ) ) {
         returnArray = underTest;
 
     } else {
-        switch( typeof underTest ) {
+        switch( Ember.typeOf( underTest ) ) {
             case 'string':
-                    returnArray = convertStringToArray( underTest );
+                returnArray = convertStringToArray( underTest );
                 break;
 
             case 'object':
```

Comment:
there are more object types beyond this one now that you have switched to using Ember.typeOf